bigchaindb_driver/driver.py

```diff
@@ -318,41 +317,6 @@ def get(self, *, asset_id, operation=None, headers=None):
             headers=headers,
         )
 
-    def send(self, transaction, mode='commit', headers=None):
-        """Submit a transaction to the Federation.
-
-        Args:
-            transaction (dict): the transaction to be sent
-                to the Federation node(s).
-            mode (str): the mode the transaction is sent to the Federation
-                node(s).
-            headers (dict): Optional headers to pass to the request.
-
-        Returns:
-            dict: The transaction sent to the Federation node(s).
-
-        .. warning::
-
-            The method .send will be deprecated in the next release
-            of the driver, please use ``.send_commit``, ``.send_sync``, or
-            ``.send_async`` instead. More info:
-            https://docs.bigchaindb.com/projects/py-driver/en/latest/handcraft.html#send-the-transaction
-        """
-
-        warn('The method .send will be deprecated in the next release of the '
```

Comment:
Does this mean that all the docs warning messages, like this

![screenshot from 2018-09-17 15-10-09](https://user-images.githubusercontent.com/1210951/45624957-082aa380-ba8c-11e8-9520-89f9871514b1.png)

will go away, or do you have to remove those manually?

Reply:
Neat.